src/Workspaces/Core/Portable/Utilities/Documentation/XmlDocumentationProvider.cs

```diff
@@ -40,7 +47,7 @@ protected override string GetDocumentationForSymbol(string documentationMemberID
                         {
                             if (e.Attribute("name") != null)
                             {
-                                _docComments[e.Attribute("name").Value] = e.Value;
+                                _docComments[e.Attribute("name").Value] = string.Concat(e.Nodes());
```

Comment:
If you're bored and want to make this more efficient, just using a XmlReader directly might be more sane here. We're using an XML reader to create a full tree, only to immediately get stuff back from it. And you could call XmlReader.ReadInnerXml which would just give you the string you want directly and avoid creating a huge pile of garbage.